codesystems/CodeSystem-England-ORDDateTime.xml

```diff
@@ -0,0 +1,28 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+	<id value="England-ORDDateTime"/>
+	<url value="https://fhir.nhs.uk/England/CodeSystem/England-ORDDateTime"/>
+	<version value="1.0.0"/>
+	<name value="EnglandORDDateTime"/>
+	<title value="England ORD Date Time"/>
+	<status value="draft" />
+	<date value="2023-10-24" />
+	<publisher value="NHS England" />
+	<contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+	</contact>
+	<description value="A set of codes that define the type of Organisation Reference Data (ORD) date and time being recorded, for the Organisation Data Service (ODS)."/>
+	<copyright value="Copyright &#169; 2023+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
```

Comment:
Copyright is out of date. Need to use the 2023+ version.
The correct one is available on Extension-England-DateTime